src/main/java/io/github/darealturtywurty/turtychemistry/core/MixinHooks.java

```diff
@@ -126,9 +126,22 @@ public static void calculateStackTemperature(final ItemStack stack, final Level
                         });
                     } else if (player.getItemBySlot(EquipmentSlot.CHEST).is(Items.LEATHER_CHESTPLATE)) {
                         player.hurt(hotIngotDamageSource, 0.5f);
+                        switch (level.getDifficulty())
+                        {
+                            case PEACEFUL -> player.hurt(hotIngotDamageSource, 0.5f);
+                            case EASY -> player.hurt(hotIngotDamageSource, (temperatureInCelsius * 0.f));
```

Comment:
`0.f`? I assume something went wrong here?

Reply:
that's weird, there was a number there when I committed it, I'll just make sure